inc/ocf_ctx.h

```diff
@@ -210,6 +210,9 @@ struct ocf_ctx_ops {
 
 	/* Logger operations */
 	struct ocf_logger_ops logger;
+
+	/* Deinitialize context */
+	void (*deinit)(void);
```

Comment:
Maybe we should just deinitialize all the volume types automatically when context is destroyed, instead of creating callback which only purpose is to perform simple operation which we can easily do internally. This only leads to introduction of boilerplate code in adapters.